os/community-platforms.md

```diff
@@ -0,0 +1,48 @@
+# Community supported platforms
+
+The CoreOS Container Linux community has provided support for Container Linux on a number of platforms beyond those [officially supported][official-support] by CoreOS.
```

Comment:
I think "The Container Linux community" reads better than "The CoreOS Container Linux community".

Reply:
ok